apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html

```diff
@@ -1,43 +1,12 @@
 {% extends "base.html" %} {% block content %}
 <div class="container">
-  <nav class="s-breadcrumbs">
-    <ol itemscope="" itemtype="https://schema.org/BreadcrumbList">
-      <li
-        itemscope=""
-        itemprop="itemListElement"
-        itemtype="https://schema.org/ListItem"
-      >
-        <a href="/" itemprop="item">
-          <span itemprop="name">Home</span>
-        </a>
-
-        <meta itemprop="position" content="1" />
-      </li>
+  {% include "nav_cms_breadcrumbs.html" %}
 
-      <li
-        itemscope=""
-        itemprop="itemListElement"
-        itemtype="https://schema.org/ListItem"
-      >
-        <a href="/register/" itemprop="item">
-          <span itemprop="name">Register</span>
-        </a>
 
-        <meta itemprop="position" content="2" />
-      </li>
-
-      <li
-        itemscope=""
-        itemprop="itemListElement"
-        itemtype="https://schema.org/ListItem"
-      >
-        <span itemprop="name">Request To Submit</span>
-
-        <meta itemprop="position" content="3" />
-      </li>
-    </ol>
-  </nav>
```

Comment:
**Explanation.** This works because app can access templates of Core-CMS's `taccsite_cms` app[^1].

**Requirement.**  To get accurate breadcrumbs, CMS must have pages and structure that match this URL.

[^1]: I don't know how, but one day I will realize how.